src/Quasar/Mount/Couchbase.purs

```diff
@@ -57,26 +64,37 @@ fromJSON
   <=< decodeJson
 
 toURI ∷ Config → URI.AbsoluteURI
-toURI { host, user, password } =
+toURI { host, bucketName, password, docTypeKey, queryTimeout } =
   URI.AbsoluteURI
     (Just uriScheme)
-    (URI.HierarchicalPart (Just (URI.Authority Nothing (pure host))) Nothing)
+    (URI.HierarchicalPart
+      (Just (URI.Authority Nothing (pure host)))
+      (Just (Right (P.rootDir </> P.file bucketName))))
     (Just (URI.Query props))
   where
   props ∷ L.List (Tuple String (Maybe String))
   props = L.Nil
-    <> maybe L.Nil (\u -> pure $ Tuple "username" (Just u)) user
-    <> maybe L.Nil (\p -> pure $ Tuple "password" (Just p)) password
+    <> pure (Tuple "password" (Just password))
+    <> pure (Tuple "docTypeKey" (Just docTypeKey))
+    <> maybe L.Nil (pure <<< Tuple "queryTimeoutSeconds" <<< Just <<< show <<< unwrap) queryTimeout
```

Comment:
i suppose my contention is

`forall a b m. Applicative m => Newtype a => Show b => a b -> m (Tuple String (Maybe String))`

is a very "loose" type.

this might not be a legitimate contention but if its not i'm interested why.